### PR TITLE
Delete Issue template

### PR DESCRIPTION
**Pull request by FeLungs**
remove issue template

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-Filing an issue about the **xAPI Profiles** specification? Please include the following
-
-- [ ] Which part/parts of the specification are at issue?
```
Comment (imartinezortiz) [inline ISSUE_TEMPLATE.md:3]: Umm maybe interesting to keep it

**Review by FeLungs — COMMENTED** [review]
example comment within review

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,9 +0,0 @@
...
-
-- [ ] What is your understanding of what the spec means in these parts?
-
-- [ ] Related to your implementation, what's the use case you're trying to achieve? What are the user stories you're trying to support?
-
```
Comment (FeLungs) [inline ISSUE_TEMPLATE.md:8]: test scoped comment

**Review by geirfp — APPROVED** [review]
Agree to remove the template :-)

**Review by imartinezortiz — APPROVED** [review]

**Pull request merged**